src/components/InputButton/InputButton.module.scss

```diff
@@ -0,0 +1,13 @@
+.container {
+  display: flex;
+  align-items: stretch;
+}
+
+.field input {
+  border-radius: 0.375rem 0 0 0.375rem;
+  min-width: 15rem;
```

Comment:
Not a huge fan of enforcing a static number, it think it should be done by the parent. I think it would be better if done this way
 - Using flex to setup the ratio (flex-grow on input and flex-shrink: 0 on button to prevent it from being squished)
 - Apply min-width on the container, but perhaps use a [clamp](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp) instead so that we have a preferred size, but not a mandatory min